src/interp/interp-inl.h

```diff
@@ -351,16 +351,40 @@ bool operator!=(const RefPtr<U>& lhs, const RefPtr<V>& rhs) {
 }
 
 //// ValueType ////
-inline bool IsReference(ValueType type) { return type.IsRef(); }
-template <> inline bool HasType<s32>(ValueType type) { return type == ValueType::I32; }
-template <> inline bool HasType<u32>(ValueType type) { return type == ValueType::I32; }
-template <> inline bool HasType<s64>(ValueType type) { return type == ValueType::I64; }
-template <> inline bool HasType<u64>(ValueType type) { return type == ValueType::I64; }
-template <> inline bool HasType<f32>(ValueType type) { return type == ValueType::F32; }
-template <> inline bool HasType<f64>(ValueType type) { return type == ValueType::F64; }
-template <> inline bool HasType<Ref>(ValueType type) { return IsReference(type); }
-
-template <typename T> void RequireType(ValueType type) {
+inline bool IsReference(ValueType type) {
```

Comment:
Wrap this `ValueType` block too?

Reply:
I ended up wrapping the whole file (otherwise I had to add so many `// clang-format off`s) so I think this will be preserved.